timeless/restaurants/tables/views.py

```diff
@@ -50,48 +34,43 @@ def list_tables():
         3: "Triangle",
     }
     return render_template(
-        "restaurants/tables/list.html", tables=tables, floors=floors,
-        shapes=shapes
+        "restaurants/tables/list.html", tables=models.Table.query.all(),
+        floors=floors, shapes=shapes
     )
 
 
 @bp.route("/create", methods=("GET", "POST"))
 def create():
-    if request.method == "POST":
-        flash("Create not yet implemented")
-    floors = [
-        {"id": 1, "name": "First location", "selected": True},
-        {"id": 2, "name": "Second location", "selected": False},
-    ]
-    shapes = [
-        {"id": 1, "name": "Rectangle", "selected": True},
-        {"id": 2, "name": "Triangle", "selected": False},
-    ]
+    """ Create new table """
+    form = forms.TableForm(request.form)
+    if request.method == "POST" and form.validate():
+        form.save()
+        return redirect(url_for("tables.list"))
     return render_template(
-        "restaurants/tables/create_edit.html", action="create",
-        floors=floors, shapes=shapes
-    )
+        "restaurants/tables/create_edit.html", form=form)
 
 
 @bp.route("/edit/<int:id>", methods=("GET", "POST"))
 def edit(id):
-    if request.method == "POST":
-        flash("Edit not yet implemented")
-    floors = [
-        {"id": 1, "name": "First location", "selected": True},
-        {"id": 2, "name": "Second location", "selected": False},
-    ]
-    shapes = [
-        {"id": 1, "name": "Rectangle", "selected": False},
-        {"id": 2, "name": "Triangle", "selected": True},
-    ]
+    """ Edit existing table """
+    table = models.Table.query.get(id)
+    if not table:
+        return redirect(url_for("tables.list"))
+
+    form = forms.TableForm(request.form, instance=table)
+    if request.method == "POST" and form.validate():
+        form.save()
+        return redirect(url_for("tables.list"))
+
     return render_template(
-        "restaurants/tables/create_edit.html", action="edit",
-        floors=floors, shapes=shapes
-    )
+        "restaurants/tables/create_edit.html", form=form)
 
 
-@bp.route("/delete", methods=["POST"])
-def delete():
-    flash("Delete not yet implemented")
+@bp.route("/delete/<int:id>", methods=["POST"])
```

Comment:
@vladarefiev Since It's template view, `DELETE` method is used mostly in RESTful APIs. You can have a look how it's made, for example, if django https://github.com/django/django/blob/master/django/views/generic/edit.py#L217